configuration/configuration.py

```diff
@@ -157,7 +157,7 @@ def _read_secret(secret_name, default = None):
 
 # The length of time (in seconds) for which a user will remain logged into the web UI before being prompted to
 # re-authenticate. (Default: 1209600 [14 days])
-LOGIN_TIMEOUT = environ.get('LOGIN_TIMEOUT', None)
+LOGIN_TIMEOUT = int(environ.get('LOGIN_TIMEOUT', None))
```

Comment:
I think this creates a problem:

```
$ python3
Python 3.9.0 (default, Oct 27 2020, 14:15:17) 
[Clang 12.0.0 (clang-1200.0.32.21)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> int(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
>>> 
```

I believe that we could change `None` to the default (at least the default according to the comment):

```suggestion
LOGIN_TIMEOUT = int(environ.get('LOGIN_TIMEOUT', 1209600))
```

Reply:
Good catch. I've added the changes.